### PR TITLE
HarfangLab EDR connector v1.2.0

### PR DESCRIPTION
**Pull request by Godefroy-Amaury**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Version 1.2.0 of HarfangLab EDR connector with a lot of new commands to fetch new telemetry, manage whitelists, get process graphs...

## Must have
- [x] Tests (unit tests + internal test suite not part of the repository)
- [x] Documentation

### CODE REVIEW

**Review by yucohen — CHANGES_REQUESTED** [review]
@Godefroy-Amaury Thanks for your contribution! Great work!
Please see my comments and add tests to the additions in the integration.
If you need any assistance don't hesitate to reach our here or on Slack.

**Comment by melamedbn** [comment]
Hi @Godefroy-Amaury,

## HarfnagLab EDR Feedback

### Layouts

#### Hurukai Threat Layout
- **Improvement**: Please enlarge the 'Linked Incidents' section to utilize the available space more effectively and avoid blank spaces in the layout.

### Incident Fields

- **Standardization Request**: To ensure consistency and avoid potential conflicts with fields from other integrations, could we prefix all incident fields related to HarfangLab EDR? For example, use 'HarfangLabEDR...' as a prefix. This would help in clearly identifying HarfangLab EDR specific fields and segregate them from other incident fields.

Thank you for addressing these enhancements to improve clarity and functionality.

Best regards,
Ben

File: Tests/known_words.txt
[the remark this answers is not in the document]
Comment (yucohen) [reply Tests/known_words.txt:1]: @Godefroy-Amaury completely makes sense, but this file is in a directory we do not accept contributions in

**Review by yucohen — CHANGES_REQUESTED** [review]

**Comment by yucohen** [comment]
Hi @Godefroy-Amaury,
Thank you for your contribution.
Your PR review might be slightly delayed due to the upcoming holiday, I apologize in advance. The holiday will be over by April 30th.
You can also try to solve the errors in the build if needed <link>.
I'm available for any of your questions here or on slack.
Thanks again.

**Review by yucohen — APPROVED** [review]

**Pull request merged**